packages/api/apprunner.yaml

```
@@ -0,0 +1,51 @@
+      - export REPO_ROOT=$(pwd)/../../
+      - cd $REPO_ROOT
+      # Install all workspace dependencies
+      - pnpm install --frozen-lockfile
+    build:
+      # Set repository root path
+      - export REPO_ROOT=$(pwd)/../../
+      # Build SDK package first (required dependency)
+      - cd $REPO_ROOT/packages/sdk
+      - pnpm build
+      # Build API package
+      - cd $REPO_ROOT/packages/api  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)